docs/data-access/data-download.md

```diff
@@ -0,0 +1,89 @@
+---
+id: data-download
+title: How to Download Data
+---
+
+The ARGO Data Platform uses the score-cient as a file download manager. Score facilitates the transfer of data with resumable download and built in BAM/CRAM slicing to make data download transfer fast and smooth.
```

Comment:
typo `cient`

Reply:
For consistency, maybe we should use the same name throughtout (ie. `Score-client` instead of `Score`)

Reply:
do we capitalize it or not do you think? `score-client` or `Score-client`?

Reply:
ive only been capitalize at sentence start and heading format

Reply:
I think lower-case should be fine. As long as it's `score-client`. I think if it's just `score`, it might look confusing.